packages/grafana-llm-app/pkg/mcp/live_server.go

```diff
@@ -148,24 +149,44 @@ type liveSession struct {
 	grafanaIdToken string
 }
 
+// tokenTimeoutSeconds is the expiration time for the Grafana Live session token.
+var tokenTimeoutSeconds = int((time.Minute * 30).Seconds())
```

Comment:
I think it would be better if this was a time.Duration and we do the slightly annoying make a new variable when we do the token exchange. That way we don't have to deal with as many unit conversions.